CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-22.8.1
+
+This releases introduces default probe authorization. This means that on
+clusters that use a default `deny` policy, probes do not have to be explicitly
+authorized using policy resources. Additionally, the
+`policyController.probeNetworks` value has been added which allows users to
+configure the networks that probes are expected to be performed from.
+
+Additionally, the `linkerd authz` command has been updated to support the policy
+resources AuthorizationPolicy and HttpRoute.
```

Comment:
should this be
```suggestion
resources `AuthorizationPolicy` and `HttpRoute`.
```

Reply:
For resources I usually default to Kubernetes docs which just worry about camel casing but not code formatting (e.g. [Pod](https://kubernetes.io/docs/concepts/workloads/pods/))

Reply:
okay, fair enough!